drizzlepac/wfpc2Data.py

```diff
@@ -164,7 +164,7 @@ def setInstrumentParameters(self, instrpars):
         # Convert the science data to electrons
         self.doUnitConversions()
 
-    def getflat(self,chip):
+    def getflat(self, chip, flat_ext=None):
```

Comment:
```suggestion
    def getflat(self, chip, flat_ext=''):
```
This would avoid the need for adding lines 179-180.  Right?

Reply:
`''` is a valid extension (i.e., no `EXTNAME` in the header) while `None` is not valid. This allows to use the default `SCI` extension for non-WFPC2 flat field files.